editor/src/components/editor/actions/actions.spec.ts

```diff
@@ -735,7 +735,7 @@ describe('moveTemplate', () => {
     expect(isUtopiaJSXComponent(updatedRoot1)).toBeTruthy()
     expect(Utils.pathOr([], ['rootElement', 'children'], updatedRoot1)).toHaveLength(1)
     const actual: any = Utils.path(['rootElement', 'children', 0], updatedRoot1)
-    expect(getLayoutPropertyOr(undefined, 'FlexFlexBasis', right(actual.props))).toBeDefined()
+    expect(getLayoutPropertyOr(undefined, 'flexBasis', right(actual.props))).toBeDefined()
     expect(getLayoutPropertyOr(undefined, 'FlexCrossBasis', right(actual.props))).toBeDefined()
```

Comment:
what about FlexCrossBasis? shouldn't we remove this guy?

Reply:
yeah, it's unused. should be removed in a cleanup PR